test/unittests/cxx20_bit_test.cpp

```diff
@@ -47,20 +47,33 @@ TEST(cxx20_bit, bit_cast_uint32_to_array)
 
 TEST(cxx20_bit, popcount32)
 {
-    EXPECT_EQ(popcount(uint32_t{0}), 0);
-    EXPECT_EQ(popcount(uint32_t{0xffffffff}), 32);
-    EXPECT_EQ(popcount(uint32_t{0xffff0000}), 16);
-    EXPECT_EQ(popcount(uint32_t{0x0000ffff}), 16);
-    EXPECT_EQ(popcount(uint32_t{0x00ffff00}), 16);
-    EXPECT_EQ(popcount(uint32_t{0x00ff00ff}), 16);
+    constexpr std::pair<uint32_t, uint32_t> test_cases[]{
```

Comment:
@chfast how about this version? Got it from `leb128_tests`.

Reply:
This is fine, but I'd put these in a header to be reused for `i32_popcnt` instruction tests.

Reply:
Done.